src/pages/mypage/profile/page.tsx

```diff
@@ -0,0 +1,107 @@
+// import React from 'react';
+
+import { useState } from 'react';
+
+export default function ProfilePage() {
+  // 자치기구 여부 확인 -> api 완료되면 수정예정
+  const [isAssociation] = useState(false);
+  return (
+    <main>
+      {/* 유저 정보 컨테이너 (자치기구, 일반학우) */}
+      {isAssociation ? (
+        <>
+          <div className="border-#D9D9D9 mb-16 ml-16 mr-10 mt-16 flex flex-row items-center rounded-lg border-2 bg-white p-10">
```

Comment:
```suggestion
          <div className="border-[#D9D9D9] my-16 ml-16 mr-10 flex items-center rounded-lg border-2 bg-white p-10">
```

`mt-*`와 `mb-*`는 `my-*`로 합칠 수 있습니다! 같은 값이라면 합치는게 보기 편해요😊

또 `flex`의 axis 기본 값은 `row`기 때문에 `flex-row`도 없어도 되지 않을까 싶습니다!

Reply:
앗 그러네요 ! 감사합니다 ㅜ ㅜ